src/test/compile-fail/macros-nonfatal-errors.rs

```diff
@@ -14,14 +14,14 @@
 #![feature(asm)]
 #![feature(trace_macros, concat_idents)]
 
-#[derive(Zero)] //~ ERROR
+#[derive(Zero)]
 struct CantDeriveThis;
```

Comment:
I'd remove this -- confusing to have erroneous code that doesn't emit errors since we abort early.